CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+- build(android): Bump Android SDK to 5.0.0-beta.2. #1508
+
+### Breaking changes
+
+- ref(android): sentry-android renamed enableSessionTracking to enableAutoSessionTracking. #1508
```

Comment:
Is this only relevant for folks not initializing the Java layer through JS?
Looks a bit alarming but without any clarifications of what actions the user should take. Do they need to change anything?